.travis.yml

```diff
@@ -2,14 +2,8 @@ language: go
 services:
   - redis-server
 go:
-  - 1.4
```

Comment:
Go 1.4 is old-ish and is losing support from modern packages. While I was there I cleaned it.